ehr-proxy/utils.py

```
@@ -0,0 +1,33 @@
+def calculate_bmi(weight_kg: float, height_m: float) -> float:
+    """Calculate Body Mass Index.
+    
+    Args:
+        weight_kg: Weight in kilograms
+        height_m: Height in meters
+        
+    Returns:
+        BMI value
+    """
+    if height_m <= 0:
+        raise ValueError("Height must be positive")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)